avm-transpiler/src/transpile.rs

```diff
@@ -384,6 +537,7 @@ fn map_brillig_pcs_to_avm_pcs(initial_offset: usize, brillig: &Brillig) -> Vec<u
 
 fn tag_from_bit_size(bit_size: u32) -> AvmTypeTag {
     match bit_size {
+        1 => AvmTypeTag::UINT8, // temp workaround
```

Comment:
brillig did not like me not having this

Reply:
I think this is fine. I do worry with temporary workarounds like this that some Noir code just won't work right in the AVM and we'll spend time debugging only to realize this. 

Link this comment to @sirasistant's [avm wishlist](https://github.com/AztecProtocol/aztec-packages/issues/4366) that mentions u1 support